platform/platform-scheduler/src/main/java/ddf/platform/scheduler/CommandJob.java

```diff
@@ -47,50 +47,76 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CommandJob.class);
 
-    private CommandProcessor commandProcessor;
+    public Subject getSystemSubject() {
+        return Security.getInstance()
+                .getSystemSubject();
```

Comment:
This will return null in the case it is not being ran as admin. Is that never the case here? If it isn't we should be wrapping with `Security.runAsAdmin`.